### PR TITLE
Unfurl links in original issue

### LINKED ISSUE #1: Unfurl links in the original issue/pr body
Currently only unfurls links in comments.

### PR DESCRIPTION
**Pull request by rsarky**
Adds functionality to unfurl links in original issue body. Fixes #1

### CODE REVIEW

**Comment by rsarky** [comment]
@bkeepers Ready for review :)

File: lib/check-unfurl-issue-comment.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    // Unfurl them into attachments
+    const unfurls = await Promise.all(links.map(link => unfurlToAttachment(link)))
+
+    context.log.trace(unfurls, 'Unfurled links')
+
+    // Attachment unfurled links to the comment
+    return attachments(context).add(unfurls)
+  }
+}
```
[the remark this answers is not in the document]
Comment (rsarky) [reply lib/check-unfurl-issue-comment.js]: That's my bad. Forgot to delete it

[the remark this answers is not in the document]
Comment (rsarky) [reply lib/check-unfurl-issue-comment.js]: > I updated the attachments extension, so you should be able to use the same code for issues, PRs, and comments now.

That's sweet. Thanks

**Comment by rsarky** [comment]
@bkeepers I have updated the code.

**Comment by bkeepers** [comment]
Thanks @rsarky! I pushed a few changes to refactor some code. I was going to also add some tests, but ran out of time and need to switch to some other work for the day.

I'm going to merge and deploy this. Thanks again for your great work here!

**Comment by rsarky** [comment]
Your welcome. @bkeepers :smile: 
I could add some tests if you want?

**Comment by bkeepers** [comment]
That'd be great!

**Pull request merged**